src/qt/peertablemodel.h

```diff
@@ -40,13 +42,15 @@ class PeerTableModel : public QAbstractTableModel
     Q_OBJECT
 
 public:
-    explicit PeerTableModel(interfaces::Node& node, QObject* parent);
+    explicit PeerTableModel(interfaces::Node& node, const PlatformStyle&, QObject* parent);
     ~PeerTableModel();
     void startAutoRefresh();
     void stopAutoRefresh();
 
+    // See also RPCConsole::ColumnWidths in rpcconsole.h
     enum ColumnIndex {
         NetNodeId = 0,
+        Direction,
```

Comment:
PR description states:
> Uses a new column so sorting is possible.

it's possible, but this PR doesn't implement it.

Because this PR doesn't implement it, the GUI will crash if you click on this column header:

```
Application Specific Information:
Assertion failed: (false), function lessThan, file qt/peertablesortproxy.cpp, line 42.
 

Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   libsystem_kernel.dylib        	0x00007fff2031e946 __pthread_kill + 10
1   libsystem_pthread.dylib       	0x00007fff2034d615 pthread_kill + 263
2   libsystem_c.dylib             	0x00007fff202a2411 abort + 120
3   libsystem_c.dylib             	0x00007fff202a17e8 __assert_rtn + 314
4   bitcoin-qt                    	0x000000010ca378d6 PeerTableSortProxy::lessThan(QModelIndex const&, QModelIndex const&) const + 918 (peertablesortproxy.cpp:42)
5   org.qt-project.QtCore         	0x000000010dfbf56e 0x10ddf9000 + 1860974
6   org.qt-project.QtCore         	0x000000010dfb3c4d 0x10ddf9000 + 1813581
7   org.qt-project.QtCore         	0x000000010dfb4308 0x10ddf9000 + 1815304
8   org.qt-project.QtCore         	0x000000
```

Here's a diff to implement sorting for this column:

``` diff
diff --git a/src/qt/peertablesortproxy.cpp b/src/qt/peertablesortproxy.cpp
index 78932da8d..dd29ddd28 100644
--- a/src/qt/peertablesortproxy.cpp
+++ b/src/qt/peertablesortproxy.cpp
@@ -24,6 +24,8 @@ bool PeerTableSortProxy::lessThan(const QModelIndex& left_index, const QModelInd
     switch (static_cast<PeerTableModel::ColumnIndex>(left_index.column())) {
     case PeerTableModel::NetNodeId:
         return left_stats.nodeid < right_stats.nodeid;
+    case PeerTableModel::Direction:
+        return left_stats.fInbound > right_stats.fInbound; // default sort Inbound, then Outbound
     case PeerTableModel::Address:
         return left_stats.addrName.compare(right_stats.addrName) < 0;
     case PeerTableModel::ConnectionType:
```

Reply:
Apparently I already fixed this but forgot to push, oops :)

(Pushed along with other fixes)